src/BrightScriptDebugSession.ts

```diff
@@ -1101,6 +1112,59 @@ export class BrightScriptDebugSession extends DebugSession {
         await Promise.all(promises);
     }
 
+    /**
+     * Will search the project files for the comment "' vscode_rale_tracker_entry" and replace it with the code needed to start the TrackerTask.
+     * @param stagingPath
+     */
+    public async injectRaleTrackerTaskCode(stagingPath: string) {
+        // Search for the tracker task entry injection point
+        let trackerEntryTerm = `('\\s*vscode_rale_tracker_entry[^\\S\\r\\n]*)`;
+        let results = Object.assign(
+            {},
+            await findInFiles.find({ term: trackerEntryTerm, flags: 'ig' }, stagingPath, /.*\.brs/),
+            await findInFiles.find({ term: trackerEntryTerm, flags: 'ig' }, stagingPath, /.*\.xml/)
+        );
+
+        let keys = Object.keys(results);
+        if (keys.length === 0) {
+            // Do not throw an error as we don't want to prevent the user from launching the channel
+            // just because they don't have a local version of the TrackerTask.
+            this.sendDebugLogLine('WARNING: Unable to find an entry point for Tracker Task.');
+            this.sendDebugLogLine('Please make sure that you have the following comment in your BrightScript project: "\' vscode_rale_tracker_entry"');
+        } else {
+            // This code will start the tracker task in the project
+            let trackerTaskSupportCode = `if true = CreateObject("roAppInfo").IsDev() then m.vs_code_tracker_task = createObject("roSGNode", "TrackerTask") ' Roku Advanced Layout Editor Support`;
```

Comment:
This is using the old `vs_code_tracker_task` but the test is not failing.

Reply:
No, what you'er looking at is the code that is going to be injected into the channel in the place of the comment. It's creating a `m.vs_code_tracker_task` variable to store the tracker in. I can change this variable to `m.vscode_rale_tracker_task` if you would like.